lib/command/maintenance.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Command
+  class Maintenance < Base
+    NAME = "maintenance"
+    OPTIONS = [
+      app_option(required: true)
+    ].freeze
+    DESCRIPTION = "Checks if maintenance mode is on or off for an app"
+    LONG_DESCRIPTION = <<~DESC
+      - Checks if maintenance mode is on or off for an app
+      - Outputs 'on' or 'off'
+      - Specify the one-off workload through `one_off_workload` in the `.controlplane/controlplane.yml` file
+      - Optionally specify the maintenance workload through `maintenance_workload` in the `.controlplane/controlplane.yml` file (defaults to 'maintenance')
+      - Maintenance mode is only supported for domains that use path based routing mode and have a route configured for the prefix '/' on either port 80 or 443
+    DESC
```

Comment:
Someday, we should consider a way to run the commands or parse the text to generate the commands.md file. Otherwise, these might be out of sync.

@rafaelgomesxyz can you search to see if somebody has already solved this problem?

Reply:
We have a task for that: `bundle exec rake command_docs`. It updates `commands.md` from the commands. I'll see if I can add a test to make sure that we don't forget to run it.